PodioKit/Common/API/PKTCalendarAPI.m

```diff
@@ -30,4 +30,72 @@ + (PKTRequest *)requestForGlobalCalendarWithFromDate:(NSDate *)fromDate toDate:(
   return [PKTRequest  GETRequestWithPath:@"/calendar/" parameters:parameters];
 }
 
++ (PKTRequest *)requestForGlobalCalendarWithFromDate:(NSDate *)fromDate toDate:(NSDate *)toDate priority:(NSUInteger)priority includeTasks:(BOOL)tasksIncluded {
+  
+  NSMutableDictionary *parameters = [NSMutableDictionary new];
+  
+  if (fromDate) {
+    parameters[@"date_from"] = [fromDate pkt_UTCDateString];
+  }
+  
+  if (toDate) {
+    parameters[@"date_to"] = [toDate pkt_UTCDateString];
+  }
+  
+  if (priority > 0) {
+    parameters[@"priority"] = @(priority);
+  }
+  
+  if (!tasksIncluded) {
+    parameters[@"tasks"] = @(NO);
+  }
+  
+  return [PKTRequest  GETRequestWithPath:@"/calendar/" parameters:parameters];
+}
+
++ (PKTRequest *)requestForAppCalendarWithAppId:(NSUInteger)appId fromDate:(NSDate *)fromDate toDate:(NSDate *)toDate priority:(NSUInteger)priority {
+  
+  NSMutableDictionary *parameters = [NSMutableDictionary new];
+  
+  if (fromDate) {
+    parameters[@"date_from"] = [fromDate pkt_UTCDateString];
+  }
+  
+  if (toDate) {
+    parameters[@"date_to"] = [toDate pkt_UTCDateString];
+  }
+  
+  if (priority > 0) {
+    parameters[@"priority"] = @(priority);
+  }
+  
```

Comment:
Ok I see this method explicitly does not take in tasks. I think we could do some refactoring here and get more code re-use. A set/build parameters method would be a good idea too